schemas/dbt/manifest/v12.json

```diff
@@ -4388,18 +4388,10 @@
                 "default": null
               },
               "primary_key": {
-                "anyOf": [
-                  {
-                    "type": "array",
-                    "items": {
-                      "type": "string"
-                    }
-                  },
-                  {
-                    "type": "null"
-                  }
-                ],
-                "default": null
+                "type": "array",
```

Comment:
Not sure what this change is - I don't think it's related to my changes, might be that someone else did not update the schema in their commit?